src/components/SegmentedControl/component.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Sections } from '../../modules/sections';
+
+import { sizes } from './styled';
+
+import { SegmentedControl } from './';
+
+export default {
+  component: SegmentedControl,
+  title: `${Sections.Elements}/SegmentedControl`,
+};
+
+export const Default = () => (
+  <>
+    {sizes.map((size) => (
+      <div key={size} style={{ marginBottom: '1em' }}>
+        <h3>{size}</h3>
+        <SegmentedControl options={['BNB', 'BTC', 'USDT']} size={size} onChange={console.log} />
```

Comment:
Maybe we could even use children instead of `options`. @essj?

Reply:
I was thinking about that but why overcomplicate, it's just strings

Reply:
Not just strings, since the default usage for `react-intl` would be passing a `<FormattedMessage />` element,.

Reply:
You can extract it into a string https://formatjs.io/docs/getting-started/message-extraction

Reply:
And what if we wanted to use an icon instead of text somewhere? Or icon + text? 🤷🏻‍♂️

Reply:
done